dissect/target/plugins/apps/remoteaccess/remoteaccess.py

```diff
@@ -7,8 +7,8 @@
     [
         ("datetime", "ts"),
         ("string", "tool"),
-        ("path", "logfile"),
         ("string", "description"),
+        ("path", "logfile"),
```

Comment:
Maybe nice to rename this field too. I think we use `source` in most other locations?